core/src/serve_repair.rs

```diff
@@ -570,6 +572,7 @@ impl ServeRepair {
             data_budget,
             cluster_type,
         );
+        stats.process_time_us += process_start.elapsed().as_micros() as u64;
```

Comment:
`handle_packets_time_us` would probably be a better name since it is wrapping `handle_packets`;
but it does not matter much.